src/main/java/Bosco.java

```diff
@@ -21,41 +22,70 @@ private static void printExitMessage() {
 
     public static void main(String[] args) {
         Scanner in = new Scanner(System.in);
-        String userInput;
+        String userInputString;
         String[] words;
```

Comment:
I think it's not appropriate to define a variable without initializing it. Perhaps you can do String words[] = userInput.split(" "); in the following code.